aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -172,6 +172,22 @@ std::vector<Tensor> chunk_batching_rule(const Tensor& self, int64_t chunks, int6
   return result;
 }
 
+std::vector<Tensor> tensor_split_sections_batching_rule(const Tensor& self, int64_t sections, int64_t dim) {
```

Comment:
@zou3519 Do you want to take a look at these changes? Would you recommend PRs start updating this file?

Reply:
This looks correct; in another comment below I suggested adding a test.

I'd recommend against suggesting PR authors in general to update this file because the developer-facing API isn't stable and I haven't written any documentation. That being said, contributions are appreciated and the below code looks good to me.